packages/console/src/pages/ApiResources/components/CreateForm/Footer.tsx

```diff
@@ -24,7 +24,7 @@ type Props = {
 function Footer({ isCreationLoading, onClickCreate }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const {
-    currentSubscription: { planId, isAddOnAvailable },
+    currentSubscription: { planId, isAddOnAvailable, isEnterprisePlan },
```

Comment:
Can we add a new variable in the context, like `isProOrEnterprisePlan`? This way, you don't need to write `planId === ReservedPlanId.Pro || isEnterprisePlan` everywhere

Reply:
For now, we can define a function here in console.